107/src/main/java/org/ehcache/jsr107/Eh107CacheManager.java

```diff
@@ -398,7 +411,7 @@ private void enableStatistics(Eh107Cache<?, ?> cache, boolean enabled) {
 
   @Override
   public boolean isClosed() {
-    return closed.get();
+    return ehCacheManager.getStatus() != Status.AVAILABLE;
```

Comment:
This should probably be == UNINITIALIZED in line with what we discussed for the Cache.